Header header
float64[9] intrinsic
float64[5] distortion
float64[9] rotation
float64[9] translation
